src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -133,8 +136,57 @@ def consent_request_verify(
     return _prepare_consent_preferences(db, provided_identity)
 
 
-@router.patch(
+@router.post(
     CONSENT_REQUEST_PREFERENCES,
+    dependencies=[Security(verify_oauth_client, scopes=[CONSENT_READ])],
+    status_code=HTTP_200_OK,
+    response_model=ConsentPreferences,
+)
+def get_consent_preferences(
+    *, db: Session = Depends(get_db), data: Identity
+) -> ConsentPreferences:
+    """Gets the consent preferences for the specified user."""
+    if data.email:
+        lookup = data.email
+    elif data.phone_number:
+        lookup = data.phone_number
+    else:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST, detail="No identity information provided"
+        )
+
+    identity = ProvidedIdentity.filter(
+        db,
+        conditions=(
+            (ProvidedIdentity.hashed_value == ProvidedIdentity.hash_value(lookup))
+            & (
+                ProvidedIdentity.privacy_request  # pylint: disable=singleton-comparison
+                == None
+            )
+        ),
+    ).first()
+
+    if not identity:
+        raise HTTPException(status_code=HTTP_404_NOT_FOUND, detail="Identity not found")
+
+    consent = Consent.filter(
+        db, conditions=(Consent.provided_identity_id == identity.id)
+    ).all()
+
+    return ConsentPreferences(
+        consent=[
+            ConsentSchema(
+                data_use=x.data_use,
+                data_use_description=x.data_use_description,
+                opt_in=x.opt_in,
+            )
+            for x in consent
+        ]
+    )
```

Comment:
You can probably use that `_prepare_consent_preferences` helper we added in the previous PR!